mmv1/third_party/terraform/services/compute/resource_compute_region_instance_template.go.erb

```diff
@@ -823,12 +823,28 @@ be from 0 to 999,999,999 inclusive.`,
 				Description: `The Confidential VM config being used by the instance. on_host_maintenance has to be set to TERMINATE or this will fail to create.`,
 				Elem: &schema.Resource{
 					Schema: map[string]*schema.Schema{
+						<% if version == "ga" %>
 						"enable_confidential_compute": {
 							Type:        schema.TypeBool,
 							Required:    true,
 							ForceNew:    true,
 							Description: `Defines whether the instance should have confidential compute enabled.`,
 						},
+						<% else %>
+						"enable_confidential_compute": {
+							Type:        schema.TypeBool,
+							Required:    true,
+							ForceNew:    true,
+							Description: `Deprecated: Use confidential_instance_type, required field to become optional. Defines whether the instance should have confidential compute enabled.`,
+						},
+						"confidential_instance_type": {
+							Type:        schema.TypeString,
+							Optional:    true,
+							Description: `
+								Specifies which confidential computing technology to use.
+								This could be one of the following values: SEV, SEV_SNP.`,
+						},
```

Comment:
```suggestion
						"enable_confidential_compute": {
							Type:        schema.TypeBool,
							Optional:    true,
							ForceNew:    true,
							Description: `Defines whether the instance should have confidential compute enabled. Field will be deprecated in a future release.`,
							AtLeastOneOf: []string{"confidential_instance_config.0.enable_confidential_compute", "confidential_instance_config.0.confidential_instance_type"},
						},
						"confidential_instance_type": {
							Type:        schema.TypeString,
							Optional:    true,
							Description: `
								Specifies which confidential computing technology to use.
								This could be one of the following values: SEV, SEV_SNP.`,
							AtLeastOneOf: []string{"confidential_instance_config.0.enable_confidential_compute", "confidential_instance_config.0.confidential_instance_type"},
						},
```